src/cloudevent/partials/alerts/alerts-on-alert-published.ts

```diff
@@ -0,0 +1,32 @@
+import {DeepPartial, MockCloudEventPartials} from '../../types';
+import {FirebaseAlertData} from 'firebase-functions/lib/v2/providers/alerts';
+import {CloudEvent, CloudFunction} from 'firebase-functions/lib/v2';
```

Comment:
just firebase-functions/v2; the actual file import isn't a module export. You might need to update your node version to fix.

Reply:
Thanks for catching that! Updated!